WeatherApp/src/main/java/org/example/LocationMenu.java

```diff
@@ -0,0 +1,40 @@
+package main.java.org.example;
+
+import java.util.Scanner;
+
+public class LocationMenu {
```

Comment:
`LocationMenu` dzieli część kodu z `Menu`, można refactorować żeby wydzielić część wspólną do jakiejś klasy nadrzędnej - ale to ma niższy priorytet.
Na razie OK.